src/hal/user_comps/wj200_vfd/wj200_vfd.comp

```diff
@@ -110,12 +116,30 @@ bool wj200_getStatus(modbus_t* ctx, wj200_status* status)
                 return false;
         }
 
+	// Read 1003h (output current) to 1019h (heatsink temp)
+        rc = modbus_read_registers(ctx, 0x1003-1, 1, currentRegister);
+
+        if(rc < 0)
+        {
+                return false;
```

Comment:
Good point. This is far from my best C code (basically copying and pasting what's already there to get the job done). As far as I know, there is only one WJ200 firmware for all WJ200 devices, so I think it's ok to assume it will always work unless there's something like a communication error or the VFD goes into fault mode.